src/components/ApplicationActionLog.tsx

```
@@ -0,0 +1,311 @@
+  ip?: string;
+  updatedAt: string;
+  status: string;
+  remark?: string;
+}
+
+interface ApplicationActionLogProps {
+  applicationData: {
+    actionLog: string[];
+    createdAt: string;
+    status?: string;
+    [key: string]: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)